dev/release-aux/openssl-announce-pre-release.tmpl

```diff
@@ -0,0 +1,49 @@
+
+   OpenSSL version $release ($LABEL)
+   ===========================================
+
+   OpenSSL - The Open Source toolkit for SSL/TLS
+   https://www.openssl.org/
+
+   OpenSSL $MAJOR.$MINOR.$PATCH is currently in $LABEL.
+
+   OpenSSL $release has now been made available.  For details of
+   changes and known issues see the release notes at:
+
+        https://www.openssl.org/news/openssl-$MAJOR.$MINOR-notes.html
+
+   Note: This OpenSSL pre-release has been provided for testing ONLY.
+   It should NOT be used for security critical purposes.
+
+   The $LABEL release is available for download via HTTPS and FTP from the
+   following master locations (you can find the various FTP mirrors under
+   https://www.openssl.org/source/mirror.html):
+
+     * https://www.openssl.org/source/
+     * ftp://ftp.openssl.org/source/
+
+   The distribution file name is:
+
+    o $tarfile
```

Comment:
I'm even unsure whether the patch number belongs in the filename for pre-releases. This comes out as:

> openssl-3.0.0-alpha1.tar.gz

But, is this better as:

> openssl-3.0-alpha1.tar.gz

Reply:
I would not recommend doing 3.0 in pre-release and switching to 3.0.0 after that. It would be confusing.

Reply:
No, but it would make sense to have openssl-3.0.tar.gz as the final release.  The patch releases that follow it would of course be openssl-3.0.1.tar.gz, openssl-3.0.2.tar.gz etc

Reply:
In theory, yes, but it would be still confusing.

Reply:
I'm sorry, what part was confusing now?

Reply:
> I'm sorry, what part was confusing now?

Never mind that...